### ISSUE TITLE
Super cool you are using the template

### ISSUE THREAD

**Issue by JavascriptMick** [CareTiger/nuxt3-saas-boilerplate#11]
Hi @CareTiger

I'm the creator of the template you used to create this site (https://github.com/JavascriptMick/supanuxt-saas) formerly known as nuxt3-boilerplate. Super pleased you have made use of the template.  You have been busy, so many commits!

Yell out if you have any questions. Note that I have added a bunch of cool features since this version including GDPR/cookie consent, rest endpoints etc. You can check out the changes here https://github.com/JavascriptMick/supanuxt-saas/blob/master/CHANGELOG.md

**Comment by CareTiger** [comment]
@JavascriptMick I started out with the best intention of using your stack but tRPC scared the shit out of me. But your project is super awesome and hardcore. I wish there was a version of it without tRPC. Also, I have recently migrated ALL new projects to NextJS  13 (app router). My clients in the US are all React/NextJS and the dev talent in India is en masse React/NextJS. I love Vue/Nuxt but.... I am really worried about the Vue/Nuxt eco system.
